clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -945,6 +945,29 @@ default ListConsumerGroupOffsetsResult listConsumerGroupOffsets(Map<String, List
         return listConsumerGroupOffsets(groupSpecs, new ListConsumerGroupOffsetsOptions());
     }
 
+    /**
```

Comment:
For all the javadocs for methods in the `Admin`, I'd suggest we add a `@note` that this effectively does the same as the corresponding consumer group method.